frontend/src/pages/PersonalData.jsx

```diff
@@ -0,0 +1,384 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { useForm } from "react-hook-form";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export default function PersonnalData() {
+  const {
+    register,
+    formState: { errors },
+  } = useForm();
+
+  const [editUser, setEditedUser] = useState(false);
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
+    axios
+      .get(
+        `${import.meta.env.VITE_BACKEND_URL ?? "http://localhost:5000"}/users/1`
+      )
+      .then((response) => response)
+      .then((data) => {
+        setUser(data.data);
+      });
+  }, []);
+
+  // Waiting for the authantification to be done so I can acces all the informations
```

Comment:
TYPO : Waiting for the **authentication** to be done so I can **access** all the information